addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -486,6 +485,145 @@ private void raiseAlert(
                 .raise();
     }
 
+    @Override
+    public List<Alert> getExampleAlerts() {
```

Comment:
It would be preferable to extract methods for each alert to avoid the need to maintain in two places for each.

Reply:
I'll have to look at it, I'm not sure how much maint that will actually save.

Reply:
It saves not forgetting to change the examples when the "main" alerts are changed, that's all.